packages/atomic/src/components/atomic-result/atomic-result-cell-desktop.pcss

```diff
@@ -34,14 +34,18 @@
       @mixin set-font-size var(--atomic-text-sm);
 
       margin-bottom: 1rem;
-      height: 2rem;
+      --line-height: 2rem;
+      line-height: var(--line-height);
+      max-height: calc(2 * var(--line-height) + 0.5rem);
     }
 
     atomic-result-section-actions {
       @mixin set-font-size var(--atomic-text-base);
 
       margin-top: 2.25rem;
-      height: 2.5rem;
+      --line-height: 2.5rem;
```

Comment:
`--line-heigh` should always be relative to font size, my previous change too care of this https://github.com/coveo/ui-kit/pull/1413

Reply:
In this case though, line height is meant to be much larger relative to the font size, since it also accounts for a background. Perhaps I should rename `--line-height` to something else? I'll rename it `row-height`.